### DISCUSSION TITLE
Seed Value "bb8\cc>~" has only one gate between 4th and 5th biomes.

### DISCUSSION THREAD

**Discussion by kaedepon** [FakeFishGames/Barotrauma#13790] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Start seed is `bb8\cc>~`
There is only one gate between 4th and 5th biomes.
I think It should be generated two gates for go to the next biome, that gates is "coalition" and "separatists".

----
![missinggate](https://github.com/FakeFishGames/Barotrauma/assets/47914478/8cd28d09-edc3-4ce9-894b-ad2c40882291)

The save data is here.
[onegate.zip](https://github.com/FakeFishGames/Barotrauma/files/15038554/onegate.zip)

### Reproduction steps

1. Input Seed `bb8\cc>~`
2. Start a campaign.
3. Watch generated map.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.3.0.4

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! This has now (finally) been [addressed in our private repo](https://github.com/FakeFishGames/Barotrauma-development/pull/8503), fix coming up in the next update.
